src/components/VerifyUser/VerifyUser.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import queryString from 'query-string';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { CardBody, CardText } from 'reactstrap';
+import propTypes from 'prop-types';
+import Card from '../Card/Card';
+import { Header } from '../Header/Header';
+import Footer from '../Footer/Footer';
+import * as actions from '../../../store/actions/authActions';
+import { Heading } from '../Heading/Heading';
+import './VerifyUser.scss';
+
+export class VerifyUser extends PureComponent {
+  componentDidMount() {
```

Comment:
Please remind me what this lifecycle hook does?

Reply:
Hello @fortunee. it runs immediately after the component mounts on the DOM.

Reply:
If this is the case, do you want to tell the user "you have been verified" in a situation where `verifyAuth`  throws an error?

@beejay1293

Reply:
@fortunee, I have refactored the above line of code,  when the component mounts, the user sees `verifying ..` on the page, then depending on what the action returns, the local state gets updates, and re-render. incase of `success`, the user would be redirected to profile page, and in case of `error`, the user would see the error message displayed on the page